L58-python-async-api.md

```diff
@@ -0,0 +1,1708 @@
+          or an grpc.aio.RpcMethodHandler if the handler is a coroutine;
+          otherwise, None.
+        """
+```
+
+```Python
+# grpc.aio.RpcMethodHandler
+class RpcMethodHandler:
+    """An implementation of a single RPC method.
+
+    Attributes:
+      request_streaming: Whether the RPC supports exactly one request message
+        or any arbitrary number of request messages.
+      response_streaming: Whether the RPC supports exactly one response message
+        or any arbitrary number of response messages.
+      request_deserializer: A callable behavior that accepts a byte string and
+        returns an object suitable to be passed to this object's business
+        logic, or None to indicate that this object's business logic should be
+        passed the raw request bytes.
+      response_serializer: A callable behavior that accepts an object produced
+        by this object's business logic and returns a byte string, or None to
+        indicate that the byte strings produced by this object's business logic
+        should be transmitted on the wire as they are.
+      unary_unary: This object's application-specific business logic as a
+        callable value that takes a request value and a ServicerContext object
+        and returns a response value. Only non-None if both request_streaming
+        and response_streaming are False.
+      unary_stream: This object's application-specific business logic as a
+        callable value that takes a request value and a ServicerContext object
+        and returns an iterator of response values. Only non-None if
+        request_streaming is False and response_streaming is True.
+      stream_unary: This object's application-specific business logic as a
+        callable value that takes an iterator of request values and a
+        ServicerContext object and returns a response value. Only non-None if
+        request_streaming is True and response_streaming is False.
+      stream_stream: This object's application-specific business logic as a
+        callable value that takes an iterator of request values and a
+        ServicerContext object and returns an iterator of response values.
+        Only non-None if request_streaming and response_streaming are both
+        True.
+    """
+    request_streaming: bool
+    response_streaming: bool
+    request_deserializer: Optional[Callable[[bytes], Any]]
+    response_serializer: Optional[Callable[[Any], bytes]]
+    unary_unary: Optional[Callable[[Any, grpc.aio.ServicerContext], Any]]
+    unary_stream: Optional[Callable[[Any, grpc.aio.ServicerContext], Optional[AsyncIterable[Any]]]]
+    stream_unary: Optional[Callable[[AsyncIterable[Any], grpc.aio.ServicerContext], Any]]
+    stream_stream: Optional[Callable[[AsyncIterable[Any], grpc.aio.ServicerContext], Optional[AsyncIterable[Any]]]]
+```
+
+```Python
+# grpc.aio.ServicerContext
+class ServicerContext(grpc.RpcContext):
```

Comment:
This is going to need an equivalent of [`add_callback()`](https://grpc.github.io/grpc/python/grpc.html#grpc.RpcContext.add_callback).